pkg/helmraiser/helm.go

```diff
@@ -12,16 +12,37 @@ import (
 
 	jsonnet "github.com/google/go-jsonnet"
 	"github.com/google/go-jsonnet/ast"
+	"github.com/grafana/tanka/pkg/kubernetes/manifest"
 	"github.com/pkg/errors"
 	yaml "gopkg.in/yaml.v3"
 )
 
-type HelmConf struct {
+// Helm provides actions on Helm charts.
+type Helm struct{}
+
+func (h Helm) run(action string, args ...string) *exec.Cmd {
```

Comment:
This is named a bit awkward, it doesn't actually run the command.

Reply:
Fixed!